app/controllers/NextUpdatesController.scala

```diff
@@ -77,8 +77,7 @@ class NextUpdatesController @Inject()(NoNextUpdatesView: NoNextUpdates,
             auditNextUpdates(user, isAgent, origin)
 
             val optOutSetup = for {
-              checks <- optOutService.getNextUpdatesQuarterlyReportingContentChecks
-              optOutOneYearViewModel <- optOutService.nextUpdatesPageOptOutViewModel()
+              (optOutOneYearViewModel, checks) <- optOutService.nextUpdatesPageOptOutWithChecks()
```

Comment:
the tuple '(optOutOneYearViewModel, checks)' can be turned to a case class named NextUpdatesOptOutViewModel, more readable and reduces arguments passed to view